WordPress/src/main/java/org/wordpress/android/viewmodel/activitylog/ActivityLogViewModel.kt

```diff
@@ -63,8 +64,8 @@ class ActivityLogViewModel @Inject constructor(
     val showRewindDialog: LiveData<ActivityLogListItem>
         get() = _showRewindDialog
 
-    private val _showActivityTypeFilterDialog = SingleLiveEvent<Unit>()
-    val showActivityTypeFilterDialog: LiveData<Unit>
+    private val _showActivityTypeFilterDialog = SingleLiveEvent<RemoteId>()
```

Comment:
IIRC we don't use `SingleLiveEvent` in new classes. Can we replace it with [Event](https://github.com/wordpress-mobile/WordPress-Android/blob/develop/WordPress/src/main/java/org/wordpress/android/viewmodel/Event.kt) class?

Reply:
This is not a new class and I wanted to stay consistent with the other fields in this class. I can replace all of the SingleLiveEvent usages in here, but we'll probably need to retest them all. I'm not sure it's worth it at the moment, wdyt?

Reply:
Ah sure, thanks for clarification.